lib/sidekiq_flow/task.rb

```diff
@@ -53,6 +53,10 @@ def skip!
 
     def clear!
       @status = STATUS_PENDING
+
+      if expired?
+        @end_date = Date.today.next_day(1).to_time.to_i
```

Comment:
Why not just `@end_date = nil` ? This is what I do usually and it works well. And it's more "expected" (POLA).
`Date.tomorrow.to_time` is equivalent and more readable, but it ends up with "2020-02-16 00:00:00 +0000" so
`Time.zone.now.tomorrow` or `1.day.from_now` is probably better.

Reply:
This gem doesn't have rails activesupport dependency hence the Date calls.

Ok will just set the end_date to nil if that is what's expected.

Reply:
Updated PR